server/src/main/java/org/candlepin/auth/KeycloakAuth.java

```diff
@@ -72,23 +90,20 @@ public KeycloakAuth(UserServiceAdapter userServiceAdapter, Provider<I18n> i18nPr
     }
 
     private void createKeycloakDeploymentFrom() {
-        try {
 
+        try {
             InputStream is = loadKeycloakConfigFile();
             kd = KeycloakDeploymentBuilder.build(is);
             deploymentContext = new AdapterDeploymentContext(kd);
-
         }
-        catch (Exception e) {
-
+        catch (FileNotFoundException e) {
+            log.error("{} :File not found", filepath);
```

Comment:
It's good that we're logging this now, but what happens if this fails? What state are we left in?

Reply:
You basically want me to add logs after the catch right?

Reply:
Not quite.

If we hit either exception here, the module's state is in a dead/failed state. Keycloak will not work and, at best, it'll silently do nothing while Candlepin keeps chugging along. Worse, it may just cause more problems when other login/request processing occurs (due to the sequential processing nature I mentioned before).

This appears to me to be a fatal exception -- one that needs to propagate up and stop Candlepin from completing normal operation until it is fixed. I could be wrong here, but if I am, it would be nice to have some documentation, comments or indication of the state CP is in and why it's okay that it's just a one-line log entry.